stock_picking_purchase_limit_days/models/product.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 PlanetaTIC - Marc Poch <mpoch@planetatic.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import date, datetime
+from dateutil.relativedelta import relativedelta 
+from odoo import models, fields, api, _
+
+
+class Product(models.Model):
+    _inherit = "product.product"
+
+    def _product_available(self, field_names=None, arg=False):
```

Comment:
IMHO this is a bit overkill to change the available computation.

Could you give a little bit explanation that leads you to do such thing ?